src/server_manager/electron_app/index.ts

```diff
@@ -66,6 +66,11 @@ function createMainWindow() {
   win.webContents.on('new-window', handleNavigation.bind(this));
   win.webContents.on('did-finish-load', () => {
     loadingWindow.hide();
+
+    // Wait until now to check for updates now so that the UI won't miss the event.
+    if (!debugMode) {
```

Comment:
Why is this changing?

Reply:
The `app-ui-ready` event that the renderer process was essentially the same thing and it means one fewer method in `preload.ts`.